package.json

```diff
@@ -10,10 +10,10 @@
     "url": "https://github.com/forgojs/forgo"
   },
   "exports": {
-    ".": "./dist/index.js",
-    "./jsx-dev-runtime": "./jsx-dev-runtime.js",
```

Comment:
These files don't exist. It looks like they're left over from the pre-2.0 `jsx-runtime` feature?